src/testpress/landing_page/index.html

```
@@ -0,0 +1,33 @@
+---
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)